packages/web3-providers-ws/test/unit/check_implementation.test.ts

```diff
@@ -0,0 +1,167 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import WebSocket from 'isomorphic-ws';
+import WebSocketProvider from '../../src/index';
+
+jest.mock('isomorphic-ws');
+// /* @ts-expect-error */
```

Comment:
```suggestion
```
I removed this one locally and didn't receive an error, but if it's needed, then I would suggest using `// @ts-expect-error`

Reply:
fixed